user_docs/en/changes.t2t

```diff
@@ -3,6 +3,12 @@ What's New in NVDA
 
 %!includeconf: ../changes.t2tconf
 
+= 2024.3 =
+
+== New Features ==
+- Added support for the BrailleEdgeS3 braille device(#16279, @EdKweon).

```

Comment:
the 2024.3 release cycle has not started yet. please move this to the other change log entry.

Reply:
Ok. I moved that to 2024.2.